### DISCUSSION TITLE
In process model retirement and webjobs sdk

### DISCUSSION THREAD

**Discussion by federiconintex** [Azure/azure-functions-durable-extension#2940] [General]
I was interested in running durable functions outside of the standard function host and I was following this article: https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-webjobs-sdk

Is this option going away given the in-process model is going to be discontinued?
Is there any other way to run durable functions within an existing project that runs the webjobs sdk?

**Comment by cgillum** [comment]
Running Durable Functions directly via the WebJobs SDK will still be supported. The difference is simply that you have to host it in your own process rather than using the Functions host (which plans to discontinue support for the in-process, WebJobs-based hosting).

As an example, Azure App Service has built-in support for apps that use the WebJobs SDK. You can use this hosting model to continue running Durable Functions via the WebJobs extension ([Microsoft.Azure.WebJobs.Extensions.DurableTask](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.DurableTask)) on App Service compute: https://learn.microsoft.com/en-us/azure/app-service/webjobs-sdk-get-started

**Reply by federiconintex** [reply]
Thanks for the clarification mate
